### ISSUE TITLE
AOT compiling error

### ISSUE THREAD

**Issue by ivanagui2** [cornflourblue/jw-angular-pagination#8]
When I try to compile with --aot flag I get the following error.

ERROR in node_modules/jw-angular-pagination/lib/jw-pagination.component.d.ts.JwPaginationComponent.html(9,10): Property 'setPage' is private and only accessible within class 'JwPaginationComponent'.
node_modules/jw-angular-pagination/lib/jw-pagination.component.d.ts.JwPaginationComponent.html(3,10): Property 'setPage' is private and only accessible within class 'JwPaginationComponent'.
node_modules/jw-angular-pagination/lib/jw-pagination.component.d.ts.JwPaginationComponent.html(6,10): Property 'setPage' is private and only accessible within class 'JwPaginationComponent'.
node_modules/jw-angular-pagination/lib/jw-pagination.component.d.ts.JwPaginationComponent.html(12,10): Property 'setPage' is private and only accessible within class 'JwPaginationComponent'.
node_modules/jw-angular-pagination/lib/jw-pagination.component.d.ts.JwPaginationComponent.html(15,10): Property 'setPage' is private and only accessible within class 'JwPaginationComponent'.

**Comment by HunteRoi** [comment]
Same with `ng build --prod`. 

Did you find a way to solve this @ivanagui2 ?

**Comment by Markuz1995** [comment]
setPage () function property must be changed from private to public

file location:
node_modules\jw-angular-pagination\lib\jw-pagination.component.d.ts

**Comment by HunteRoi** [comment]
Yeah we figured that out. Work around would be to extract the component and put it within your application. The point of this issue is currently to make it fixed in the library itself.

**Comment by hamdirhibi** [comment]
it's work with me after deleting private notation in 'setPage' method in node_modules/jw-angular-pagination/lib/jw-pagination.component.d.ts
